### PR TITLE
Cleanup 30/11/2019

### LINKED ISSUE #1167: Missing icons
```
## WARNING: design /datum/design/bottle/export/minikeg with icon 'icons/obj/drinks.dmi' missing state 'keggy'
## WARNING: design /datum/design/bottle/export/blooddrop with icon 'icons/obj/drinks.dmi' missing state 'champagne_selling_bottle'
## WARNING: design /datum/design/bottle/export/slim_gold with icon 'icons/obj/drinks.dmi' missing state 'selling_bottle_alt'
## WARNING: design /datum/design/bottle/export/white_bloodmoon with icon 'icons/obj/drinks.dmi' missing state 'selling_bottle_basic'
## WARNING: design /datum/design/bottle/export/greenroad with icon 'icons/obj/drinks.dmi' missing state 'selling_bottle'
```
I blame the mirror bot

### LINKED ISSUE #1173: No costume vender on Box
Issue reported from  Round ID: 193 (KeplerStation)

Reporting client version: 512
Tittle, this was super close to round start so I don't think anyone took it.
unknown

### LINKED ISSUE #1175: Medical beret has no sprite
It has no wearing or item sprite.

image

image

### LINKED ISSUE #1176: fairygrass (harvested plant) has no sprite
## Reproduction: grow fairygrass (mutate grass), harvest.

EDIT: APPLIES TO JUPITER CUPS (mutate chanterelle), and the seeds.

### PR DESCRIPTION
**Pull request by AffectedArc07**
## About The Pull Request
Fixes some issues which were caused by the mirrorbot, as well as a few renames

Fixes #1176
Fixes #1175
Fixes #1173
Fixes #1167

## Why It's Good For The Game
We should have icons for stuff, and shouldnt have nanotrasen

## Changelog
:cl:
tweak: Converted more NT to Horizons
add: Added an autodrobe to BoxStation locker room
imageadd: Added images for the harvest sprite for fairygrass
imageadd: Added images for missing berets
imageadd: Added images for bottle exports
/:cl:

### CODE REVIEW

File: code/modules/awaymissions/mission_code/snowdin.dm [hunk 1]
```diff
@@ -373,7 +373,7 @@
 	PRESET /datum/preset_holoimage/captain
 	NAME Caleb Reed
 	DELAY 10
-	SAY Paid in experience! That's the Nanotrasen Motto!
+	SAY Paid in experience! That's the Horizons Motto!
```
Comment (Shadeykins) [inline code/modules/awaymissions/mission_code/snowdin.dm]: Paid in adventure! That's the Horizons motto!

File: code/modules/awaymissions/mission_code/snowdin.dm [hunk 2]
```diff
@@ -386,7 +386,7 @@
 	DELAY 30
 	SAY You have been selected out of $)@! potential candidates for this post!
 	DELAY 30
-	SAY Nanotrasen is pleased to have you working in one of the many top-of-the-line research posts within the $%@!! sector!
+	SAY Horizons is pleased to have you working in one of the many top-of-the-line research posts within the $%@!! sector!
```
Comment (Shadeykins) [inline code/modules/awaymissions/mission_code/snowdin.dm]: SAY Horizons is pleased to have you working in one of the many exploratory posts within the $%@!! sector!

File: code/modules/cargo/packs/costumes_toys.dm [hunk 3]
```diff
@@ -280,7 +280,7 @@
 
 /datum/supply_pack/costumes_toys/wizard
 	name = "Wizard Costume Crate"
-	desc = "Pretend to join the Wizard Federation with this full wizard outfit! Nanotrasen would like to remind its employees that actually joining the Wizard Federation is subject to termination of job and life."
+	desc = "Pretend to join the Wizard Federation with this full wizard outfit! Head Office would like to remind its employees that actually joining the Wizard Federation is subject to termination of job and life."
```
Comment (Shadeykins) [inline code/modules/cargo/packs/costumes_toys.dm]: Pretend to join the Wizard Federation with this full wizard outfit! Head Office would like to remind all employees that actually joining the Wizard Federation is subject to termination of job and life as per clause 47(c) of the ICC's magic and wizardry accord.

File: code/modules/cargo/packs/engineering.dm [hunk 4]
```diff
@@ -12,7 +12,7 @@
 
 /datum/supply_pack/engineering/shieldgen
 	name = "Anti-breach Shield Projector Crate"
-	desc = "Hull breaches again? Say no more with the Nanotrasen Anti-Breach Shield Projector! Uses forcefield technology to keep the air in, and the space out. Contains two shield projectors."
+	desc = "Hull breaches again? Say no more with the Horizons Anti-Breach Shield Projector! Uses forcefield technology to keep the air in, and the space out. Contains two shield projectors."
```
Comment (Shadeykins) [inline code/modules/cargo/packs/engineering.dm]: Hull breaches again? Say no more with the AccuTech Anti-Breach Shield Projector! Uses forcefield technology to keep the air in, and the space out. Contains two shield projectors.

**Comment by AffectedArc07** [comment]
@Shadeykins Issues rectified

**Pull request merged**